### PR TITLE
[Doc][Resources] Instruct users to use local storage if they have remote storage mounted to local

### LINKED ISSUE #11427: [Doc][Resources] Instruct users to use local storage if they have remote storage mounted to local
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

* Local storage configuration of `resource center` is more than `for a single machine`. If users have their remote storage mounted to local, they could use 'local storage' configuration to operate stuff in their remote storage.

### Documentation Links

* https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/guide/resource/configuration.html

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EricGao888**
## Purpose of the pull request

* Instruct users to use local storage if they have remote storage mounted to local.
* This PR closes: #11427 

## Brief change log

* Instruct users to use local storage if they have remote storage mounted to local.
* Fix bugs in resource center configuration docs.
* Polish the related docs.

## Verify this pull request

* This pull request is doc improvement without any test coverage.

### CODE REVIEW

File: docs/docs/en/guide/resource/configuration.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
-# HDFS Resource Configuration
+# Resource Center Configuration
 
-When it is necessary to use the Resource Center to create or upload relevant files, all files and resources will be stored on HDFS. Therefore the following configuration is required.
+- You could use `Resource Center` to upload text files, UDFs and other task-related files.
+- You could configure `Resource Center` to use distributed file system like `HDFS` (2.6+), `MinIO` or remote storage products like `AWS S3`, `Alibaba Cloud OSS`, etc.
+- You could configure `Resource Center` to use local file system. If you deploy `DolphinScheduler` in `Standalone` mode, you could configure it to use local file system for `Resouce Center` without the need of an external `HDFS` system or `S3`.
+- Furthermore, if you deploy `DolphinScheduler` in `Cluster` mode, you could use `S3FS-FUSE` to mount `S3` or `Jindo-FUSE` to mount `OSS` to your machines and use the local file system for `Resouce Center`. In this way, you could operate remote files as if on your local machines.
```
Comment (zhongjiajie) [inline docs/docs/en/guide/resource/configuration.md]: can we add some links and remove ` for the example file system? like [MinIO](https://min.io)

**Review by SbloodyS — DISMISSED** [review]
LGTM

File: docs/docs/en/guide/resource/configuration.md [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
 
-For a single machine, you can choose to use local file directory as the upload directory (no need to deploy Hadoop) by making the following configuration.
+### Configure `common.properties`
 
-### Configuring the `common.properties`
+Configure `api-server/conf/common.properties` and `worker-server/conf/common.properties` as follows:
 
-Configure the file in the following paths: `api-server/conf/common.properties` and `worker-server/conf/common.properties`.
+- Change `resource.storage.upload.base.path` to your local directory path. Please make sure the `tenant resource.hdfs.root.user` has read and write permissions for `resource.storage.upload.base.path`, e,g. `/tmp/storage/dolphinscheduler`. `DolphinScheduler` will create the directory you configure if it does not exist.
+- Modify `resource.storage.type=HDFS` and `resource.hdfs.fs.defaultFS=file:///`.
```
Comment (zhongjiajie) [inline docs/docs/en/guide/resource/configuration.md:15]: Should we add a notice to tell users to modify `resource.storage.upload.base.path` if they want to change the base resource path?

```markdown
> NOTE: balabalabala
```

File: dolphinscheduler-python/pydolphinscheduler/README.md [hunk 3]
```diff
@@ -84,4 +84,4 @@ If you are interested in how to release **PyDolphinScheduler**, you could go and
 
 ## What's more
 
-For more detail information, please go to see **PyDolphinScheduler** [document](https://dolphinscheduler.apache.org/python/index.html)
+For more detail information, please go to see **PyDolphinScheduler** [document](https://dolphinscheduler.apache.org/python/3.0.0/index.html)
```
Comment (zhongjiajie) [inline dolphinscheduler-python/pydolphinscheduler/README.md]: Good catch, I also add in https://github.com/apache/dolphinscheduler/pull/11170/files#diff-b26d7dd07b6c97abef7a204c696762ad9e04fc24713c3817d6ecb22fb84986f3

File: docs/docs/en/guide/resource/configuration.md (hunk 1 above)
Comment (EricGao888) [reply docs/docs/en/guide/resource/configuration.md]: Sure, good idea

File: docs/docs/en/guide/resource/configuration.md (hunk 2 above)
Comment (EricGao888) [reply docs/docs/en/guide/resource/configuration.md:15]: Yes, will add it in the next commit

File: dolphinscheduler-python/pydolphinscheduler/README.md (hunk 3 above)
Comment (EricGao888) [reply dolphinscheduler-python/pydolphinscheduler/README.md]: Thx for the information. I will change `3.0.0` to `dev` or you could merge your PR and I will rebase on it.

**Review by zhongjiajie — DISMISSED** [review]

**Review by zhongjiajie — APPROVED** [review]

**Pull request merged**